PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalRequest.kt

```diff
@@ -72,6 +72,8 @@ import org.json.JSONException
  * where the user has a PayPal Account with the same email.
  * @property userPhoneNumber User phone number used to initiate a quicker authentication flow in
  * cases where the user has a PayPal Account with the phone number.
+ * @property shopperSessionId value should be the shopper session ID returned from your server SDK
```

Comment:
I think this doc string was included in the ticket but I'm wondering if it's clear enough for when we are outside of the shopper insights module. Should we include something like `the shopper session ID returned from your shopper insights server SDK integration?`. Also should this param be beta/experimental API @jaxdesmarais?

Reply:
I like it. 
Current: 
shopperSessionId value should be the shopper session ID returned from your server SDK request.
To:
shopperSessionId  the shopper session ID returned from your shopper insights server SDK integration?

Reply:
I'll make the changes in iOS as well.

Reply:
Yep we can for sure give this the beta/experimental API tag